Scheduler/observing_blocks_auxtel/BLOCK-296/BLOCK-296_g.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "BLOCK-296_g",
+  "id": "99d226d3-f447-449f-a872-de15db82bc23",
+  "program": "BLOCK-296",
+  "constraints": [],
+  "scripts": [
+    {
+      "name": "auxtel/track_target.py",
+      "standard": true,
+      "parameters": {
+        "find_target": {
```

Comment:
I'm a little worried you might not find targets at all positions. Could you increase the radius to 5 degrees and add a "mag_range": 4.0? That would mean you would be searching for targets between 8-12mag. The risk is that if you can't find a target at one position, the whole block will need to be repeated or queued up manually.